source/unified-test-format/schema-1.5.json

```diff
@@ -190,7 +190,22 @@
       }
     },
 
-    "collectionData": {
+    "initialCollectionData": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["collectionName", "databaseName", "documents"],
+      "properties": {
+        "collectionName": { "type": "string" },
+        "databaseName": { "type": "string" },
+        "collectionOptions": { "type": "object" },
```

Comment:
Shouldn't this reference `collectionOrDatabaseOptions`?

Reply:
Thanks. I meant to delete this comment before submitting the review, as I answered the question for myself when I read further along in the PR (see my other comment about advising against using `writeConcern`).